packages/docs/docs/changelog.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 0.7.2
+## 0.7.3
+
+- Update readme on npm.
```

Comment:
```suggestion
- Update readme.
```

It's not just updated on npm.

Reply:
the readme on github was updated separately so this change actually only changes on npm.